SYCL/lit.cfg.py

```diff
@@ -49,7 +49,6 @@
 elif platform.system() == "Windows":
     config.available_features.add('windows')
     llvm_config.with_system_environment('LIB')
-    llvm_config.with_environment('LIB', config.sycl_libs_dir, append_path=True)
```

Comment:
Could you please clarify why this line is being removed?

Reply:
> Could you please clarify why this line is being removed?

LIB environment variable is used for build time (should contain dir where *.lib files are stored) and we do not need it to point to bin directory where *.dll files are stored. I will remove LIB here as all.